### PR TITLE
Added System.IO package to resolve IOException/FileNotFound error

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: src/Nancy/project.json [hunk 1]
```diff
@@ -19,12 +20,18 @@
             "System.Data.Common": "4.0.1-beta-23516",
             "System.Diagnostics.Tools": "4.0.1-beta-23516",
             "System.Dynamic.Runtime": "4.0.11-beta-23516",
+            "System.IO": "4.0.11-beta-23516",
             "System.IO.FileSystem": "4.0.1-beta-23516",
             "System.Linq": "4.0.1-beta-23516",
             "System.Net.NameResolution": "4.0.0-beta-23516",
             "System.Net.Primitives": "4.0.11-beta-23516",
+<<<<<<< Updated upstream
```
Comment (khellang) [inline src/Nancy/project.json]: What?

File: src/Nancy/project.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "licenseUrl": "https://github.com/NancyFx/Nancy/blob/master/license.txt",
     "iconUrl": "http://nancyfx.org/nancy-nuget.png",
 
+
```
Comment (khellang) [inline src/Nancy/project.json]: :fire:

Comment (jchannon) [inline src/Nancy/project.json]: Huh?

Comment (khellang) [inline src/Nancy/project.json]: Two blank lines in a row...

uh uh

**Pull request merged**